### PR TITLE
Feature/webbrowser update

### PR DESCRIPTION
**Pull request by martinfalk**
* added system settings for adjusting the refresh rate (default 30Hz)
* fixed occasional crashes when using the mouse wheel
* updated CEF which required minor fixes (https://bitbucket.org/chromiumembedded/cef/issues/3185/inconsistent-namespace-usage-in)

### CODE REVIEW

**Review by petersteneteg — DISMISSED** [review]
lgtm

File: modules/webbrowser/include/modules/webbrowser/webbrowsersettings.h [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#include <inviwo/core/util/settings/settings.h>
+#include <inviwo/core/properties/ordinalproperty.h>
+
+namespace inviwo {
+
+class IVW_MODULE_WEBBROWSER_API WebBrowserSettings : public Settings {
+public:
+    WebBrowserSettings();
+
+    IntProperty refreshrate_;
```
Comment (ResearchDaniel) [inline modules/webbrowser/include/modules/webbrowser/webbrowsersettings.h]: Camel case typo
refreshRate_

File: modules/webbrowser/src/webbrowsermodule.cpp [hunk 2]
```diff
@@ -112,6 +113,14 @@ WebBrowserModule::WebBrowserModule(InviwoApplication* app)
     // Call 60 times per second
     , doChromiumWork_(Timer::Milliseconds(1000 / 60), []() { CefDoMessageLoopWork(); }) {
```
Comment (ResearchDaniel) [inline modules/webbrowser/src/webbrowsermodule.cpp]: Don’t you need to use the WebbrowserSettings default value here?

File: modules/webbrowser/src/webbrowsersettings.cpp [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+ *
+ *********************************************************************************/
+
+#include <modules/webbrowser/webbrowsersettings.h>
+
+namespace inviwo {
+
+WebBrowserSettings::WebBrowserSettings()
+    : Settings("Web Browser")
+    , refreshrate_("refreshrate", "Refresh Rate (Hz)", 30, {0, ConstraintBehavior::Immutable},
```
Comment (ResearchDaniel) [inline modules/webbrowser/src/webbrowsersettings.cpp]: Either you need to treat 0 as a special case or set minimum to 1. 0 result in infinite value in the Webbrowser module

**Review by ResearchDaniel — CHANGES_REQUESTED** [review]
Nice! Some minor things that need to be addressed. I would also like to test the branch before it is merged. Will do that ASAP

File: modules/webbrowser/src/webbrowsermodule.cpp (hunk 2 above)
Comment (martinfalk) [reply modules/webbrowser/src/webbrowsermodule.cpp]: sure, forgot about to change it here.

File: modules/webbrowser/src/webbrowsersettings.cpp (hunk 3 above)
Comment (martinfalk) [reply modules/webbrowser/src/webbrowsersettings.cpp]: Good catch. Was also considering refresh rates below 1 second, but then one would need to use a float property. Not sure how useful that would be.

**Review by ResearchDaniel — APPROVED** [review]
Also tried it on my computer without issues

**Pull request merged**